tests/tests/connectors/test-class-connector-acf.php

```diff
@@ -0,0 +1,193 @@
+<?php
+namespace WP_Stream;
+
+class Test_WP_Stream_Connector_ACF extends WP_StreamTestCase {
+	protected $group_key;
+	protected $field_key;
+
+
+	public function setUp() {
+		parent::setUp();
+
+		// Make partial of Connector_ACF class, with mocked "log" function.
+		$this->mock = $this->getMockBuilder( Connector_ACF::class )
+			->setMethods( [ 'log' ] )
+			->getMock();
+
+		// Register connector.
+		$this->mock->register();
+
+		// Reusable ACF object keys.
+		$this->group_key = __CLASS__;
+		$this->field_key = 'field_5d7812fd000a4';
+	}
+
+	public function tearDown() {
+		parent::tearDown();
+	}
+
+	public function update_acf_field_group( $config = [] ) {
+		$defaults = [
+			'key'                   => $this->group_key,
+			'title'                 => 'Test Group',
+			'fields'                => [],
+			'location'              => [
+				[
+					[
+						'param'    => 'post_type',
+						'operator' => '==',
+						'value'    => 'post',
+					],
+				],
+			],
+			'menu_order'            => 0,
+			'position'              => 'normal',
+			'style'                 => 'default',
+			'label_placement'       => 'top',
+			'instruction_placement' => 'label',
+			'hide_on_screen'        => '',
+			'active'                => true,
+			'description'           => '',
+		];
+
+		return \acf_update_field_group( array_merge( $defaults, $config ) );
+	}
+
+	public function update_acf_field( $config = [] ) {
+		$defaults = [
+			'parent'            => $this->group_key,
+			'key'               => $this->field_key,
+			'label'             => 'Test Field',
+			'name'              => 'test_field',
+			'type'              => 'text',
+			'instructions'      => '',
+			'required'          => 0,
+			'conditional_logic' => 0,
+			'wrapper'           => array(
```

Comment:
Sometimes you are using shorthand arrays and other times using the long form of array. We need to pick one an stick with that form throughout the plugin.

Reply:
For now, I go with long version because it used through the plugin.